pkg/cloudprovider/vsphere/loadbalancer/config/config_test.go

```diff
@@ -0,0 +1,103 @@
+/*
+ Copyright 2020 The Kubernetes Authors.
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+     http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+*/
+
+package config
+
+import (
+	"fmt"
+	"strings"
+	"testing"
+)
+
+func TestReadConfig(t *testing.T) {
+	s1 := `
+[LoadBalancer]
+ipPoolName = pool1
+size = MEDIUM
+lbServiceId = 4711
+logicalRouterId = 1234
+
+[LoadBalancerClass "public"]
+ipPoolName = poolPublic
+
+[LoadBalancerClass "private"]
+ipPoolName = poolPrivate
+
+[Tags]
```

Comment:
This name is too generic if it's for LB tagging only.

Reply:
May be those tags can be used for all vSphere/NSXT-T elements that are generated by the controller manager to be able to identify those elements later o (or by a human being using the GUI). So far, only the LB controller creates such elements. So  it is the only controller consuming this tags section. But nevertheless it should have a more common name, because of it's intended general usage.

Reply:
We've moved it to the `LoadBalancer` section now